public/web_worker/patchBuildVariantsReduce.js

```diff
@@ -1,37 +0,0 @@
-// Iterate through PatchBuildVariants and determine if a task should be
```

Comment:
The webworker helped with checkbox preselection. Since calculation for checkbox states can be expensive, the webworker performs the calculation in a separate thread to prevent user interaction and loading icon lag when selecting statuses in the Tasks Table. We should adjust this depending on what we decide to do about checkbox preselection in the restart modal.

Reply:
I had made the suggestion to remove it/ not include it in the refactor. I think it may have been broken for a while and we haven't noticed it. It has been throwing bugsnag errors for a while now.  https://app.bugsnag.com/mongodb/evergreen/errors/5fe38df14daa980017d77628?filters[event.since]=30d&filters[error.status]=open

Reply:
The bugsnag error says that there is a js syntax error coming from the file but I don't think the error report indicates an issue. I can tell that the webworker is valid JS and works because I'm able to see the correct output from the webworker after setting a breakpoint from console -> sources and searching for `patchBuildVariantsReduce.js`. I tested this with mozilla on the same URL from the bugsnag report.